prime-router/src/main/kotlin/ActionMessages.kt

```diff
@@ -169,4 +169,12 @@ class DuplicateItemMessage() : ActionLogDetail {
  */
 class FhirActionLogDetail(
     override val message: String
-) : GenericActionLogDetail(message, ActionLogScope.report)
\ No newline at end of file
+) : GenericActionLogDetail(message, ActionLogScope.report)
+
+/**
+ * A [message] for invalid processing type
+ */
+class UnsupportedProcessingTypeMessage() : ActionLogDetail {
+    override val scope = ActionLogScope.report
+    override val message = "Full ELR senders must be configured for async processing."
```

Comment:
Why provide this message?  Can we just always process them async?

Reply:
We give the option for a sender to be set as 'sync' or for an override processing mode 'sync' to be sent in. Doing so suggests the client will receive a 'destinations' block back as part of their initial send. If we allow them to think they are processing sync but then do it async, there could be confusion.